testsuite/integration/clustering/src/test/java/org/jboss/as/test/clustering/cluster/xsite/XSiteSimpleTestCase.java

```diff
@@ -198,58 +198,85 @@ public void test(@ArquillianResource(CacheAccessServlet.class) @OperateOnDeploym
-                    .teardown("/subsystem=jgroups/stack=tcp/protocol=TCPPING:write-attribute(name=socket-bindings,value=[node-1,node-2,node-3,node-4])")
-                    .teardown("/subsystem=jgroups/stack=tcp/relay=RELAY:remove")
-                    .teardown("/subsystem=jgroups/channel=bridge:remove")
-                    .teardown("/subsystem=infinispan/cache-container=foo/distributed-cache=bar/component=backups/backup=SFO:remove")
-                    .teardown("/subsystem=infinispan/cache-container=foo/distributed-cache=bar/component=backups/backup=NYC:remove")
-                    .parent()
-                    // NYC
-                    .node(NODE_3)
-                    .setup("/subsystem=jgroups/channel=bridge:add(stack=tcp-bridge)")
-                    .setup("/subsystem=jgroups/stack=tcp/relay=RELAY:add(site=NYC)")
-                    .setup("/subsystem=jgroups/stack=tcp/relay=RELAY/remote-site=LON:add(channel=bridge)")
-                    .setup("/subsystem=jgroups/stack=tcp/relay=RELAY/remote-site=SFO:add(channel=bridge)")
-                    .setup("/subsystem=jgroups/stack=tcp/protocol=TCPPING:write-attribute(name=socket-bindings,value=[node-3])")
-                    .teardown("/subsystem=jgroups/stack=tcp/protocol=TCPPING:write-attribute(name=socket-bindings,value=[node-1,node-2,node-3,node-4])")
-                    .teardown("/subsystem=jgroups/stack=tcp/relay=RELAY:remove")
-                    .teardown("/subsystem=jgroups/channel=bridge:remove")
-                    .parent()
-                    // SFO
-                    .node(NODE_4)
-                    .setup("/subsystem=jgroups/channel=bridge:add(stack=tcp-bridge)")
-                    .setup("/subsystem=jgroups/stack=tcp/relay=RELAY:add(site=SFO)")
-                    .setup("/subsystem=jgroups/stack=tcp/relay=RELAY/remote-site=LON:add(channel=bridge)")
-                    .setup("/subsystem=jgroups/stack=tcp/relay=RELAY/remote-site=NYC:add(channel=bridge)")
-                    .setup("/subsystem=jgroups/stack=tcp/protocol=TCPPING:write-attribute(name=socket-bindings,value=[node-4])")
-                    .teardown("/subsystem=jgroups/stack=tcp/protocol=TCPPING:write-attribute(name=socket-bindings,value=[node-1,node-2,node-3,node-4])")
-                    .teardown("/subsystem=jgroups/stack=tcp/relay=RELAY:remove")
-                    .teardown("/subsystem=jgroups/channel=bridge:remove")
-            ;
+            super(createContainerSetConfigurationBuilder()
+                    .addContainers(NODE_1_2, createContainerConfigurationBuilder()
+                        .setupScript(createScriptBuilder()
+                            .startBatch()
+                                .add("/subsystem=infinispan/cache-container=foo/distributed-cache=bar/component=backups/backup=NYC:add(failure-policy=WARN,strategy=SYNC,timeout=10000,enabled=true)")
```

Comment:
We have discussed that we are going to be using spaces after commas delimiting parameters in a CLI operation since it's more readable. So updating `CLIServerSetupTask` to `ManagementServerSetupTask` is a great time to fix.

```
.add("/subsystem=infinispan/cache-container=foo/distributed-cache=bar/component=backups/backup=NYC:add(failure-policy=WARN, strategy=SYNC, timeout=10000, enabled=true)")
```